sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedIterable.java

```diff
@@ -23,18 +23,18 @@
  *
  * {@codesnippet com.azure.core.http.rest.pagedIterable.iterableByPage.while}
```

Comment:
Code snippets should include examples for `PagedFlux` and `CustomPagedFlux` extending from `PagedFluxBase`.

Reply:
Added